### DISCUSSION TITLE
Questions about Kong OSS update frequency

### DISCUSSION THREAD

**Discussion by mfbmina** [Kong/kong#13625] [Help] [answered]
Hey guys, 

Do you have a regular frequency of updates on Kong OSS? For example, every month there is a new minor version and every year there is a new major version.

If not, what is the easiest way to discover new versions?

**Comment by chronolaw** [comment, marked as answer]
We will release a new minor version about each three month, but for major version it has no promise.

**Comment by chronolaw** [comment]
You could find the latest version either in github(https://github.com/Kong/kong/releases) or docker hub (https://hub.docker.com/_/kong/).

**Reply by mfbmina** [reply]
Thanks @chronolaw ! 👍🏼
